docs/tutorial/graphql.rst

```diff
@@ -0,0 +1,185 @@
+.. _ref_tutorial_graphql:
+
+GraphQL
+=======
+
+In order to set up GraphQL access to the database we need to update the
+configuration:
+
+.. code-block:: edgeql-repl
+
+    tutorial> CONFIGURE SYSTEM INSERT Port {
+    .........     protocol := "graphql+http",
+    .........     database := "tutorial",
+    .........     address := "127.0.0.1",
+    .........     port := 8888,
+    .........     user := "http",
+    .........     concurrency := 4,
+    ......... };
+    CONFIGURE SYSTEM
+
+This will expose :ref:`GraphQL API <ref_graphql_index>` on port 8888.
+Pointing your browser to ``http://127.0.0.1:8888/explore`` will bring
+up a GraphiQL interface to EdgeDB. This interface can be used to try
+out queries and explore the GraphQL capabilities.
+
+Let's look at a basic ``Movie`` query:
+
+.. code-block:: graphql
+
+    {
+        Movie {
+            title
+            year
+        }
+    }
+
+Which results in:
+
+.. code-block:: json
+
+    {
+      "data": {
+        "Movie": [
+          {
+            "title": "Blade Runner 2049",
+            "year": 2017
+          },
+          {
+            "title": "Dune",
+            "year": null
+          }
+        ]
+      }
+    }
+
+It's possible apply a :ref:`filter <ref_graphql_overview_filter>` to
```

Comment:
"It's possible apply" -> "It's possible **to** apply" ?

Reply:
OK